upup/pkg/kutil/convert_kubeup_cluster.go

```diff
@@ -480,7 +480,7 @@ func (x *ConvertKubeupCluster) Upgrade() error {
 		return fmt.Errorf("error writing completed cluster spec: %v", err)
 	}
 
-	oldCACertPool, err := oldKeyStore.CertificatePool(fi.CertificateId_CA, true)
+	oldCACertPool, err := oldKeyStore.FindCertificatePool(fi.CertificateId_CA)
```

Comment:
This one isn't obviously safe (because FindCertificatePool won't create the certificate if it wasn't found.)  Split it out into a new PR?